concent_api/common/tests/test_api_view.py

```diff
@@ -271,20 +303,32 @@ def test_api_view_should_rollback_changes_on_500_error(self):
                         PROVIDER_PRIVATE_KEY,
                         CONCENT_PUBLIC_KEY
                     ),
-                    content_type                        = 'application/octet-stream',
+                    content_type='application/octet-stream',
                 )
-            except TypeError:
+            except CustomException:
```

Comment:
please change:
```python
try:
   # some code here
except CustomException:
   pass
```
to
```python
with suppres(CustomException):
   # some code here
```
throughout the code

Reply:
Why not `self.assertRaises(CustomException)` ?